components/table/Table.tsx

```diff
@@ -87,7 +90,65 @@ const defaultPagination = {
  */
 const emptyObject = {};
 
-export default class Table<T> extends React.Component<TableProps<T>, TableState<T>> {
+let tableRow: BodyRowClass | undefined;
+
+const createComponents = (
+  components: TableComponents = {},
+  prevComponents?: TableComponents,
+  isCalledFromConstructor?: boolean,
+) => {
+  const bodyRow = components && components.body && components.body.row;
+  const prevBodyRow = prevComponents && prevComponents.body && prevComponents.body.row;
+  if (isCalledFromConstructor) {
+    // 'tableRow' needs to act like a class property
+    tableRow = undefined;
```

Comment:
I don't quite understand why adding this, what I think `tableRow` is `undefined` in constructor already. Any other where the `tableRow` had been used?

Reply:
@kristof0425

Reply:
https://github.com/ant-design/ant-design/pull/18953/commits/a5ad47b4870af0d510798ba0193fdcf9eb102265